recipes/cassandra-cpp-driver/all/conanfile.py

```diff
@@ -0,0 +1,145 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class CassandraCppDriverConan(ConanFile):
+    name = "cassandra-cpp-driver"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://docs.datastax.com/en/developer/cpp-driver/"
+    description = "DataStax C/C++ Driver for Apache Cassandra and DataStax Products"
+    topics = ("cassandra", "cpp-driver", "database", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "install_header_in_subdir": [True, False],
+        "use_atomic": [None, "boost", "std"],
+        "with_openssl": [True, False],
+        "with_zlib": [True, False],
+        "with_kerberos": [True, False],
+        "use_timerfd": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "install_header_in_subdir": False,
+        "use_atomic": None,
+        "with_openssl": True,
+        "with_zlib": True,
+        "with_kerberos": False,
+        "use_timerfd": True
+    }
+
+    generators = "cmake"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/*"
+    ]
+
+    _cmake = None
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+            
+        self._cmake = CMake(self)
+        self._cmake.definitions["VERSION"] = self.version
+        self._cmake.definitions["CASS_BUILD_EXAMPLES"] = False
+        self._cmake.definitions["CASS_BUILD_INTEGRATION_TESTS"] = False
+        self._cmake.definitions["CASS_BUILD_SHARED"] = self.options.shared
+        self._cmake.definitions["CASS_BUILD_STATIC"] = not self.options.shared
+        self._cmake.definitions["CASS_BUILD_TESTS"] = False
+        self._cmake.definitions["CASS_BUILD_UNIT_TESTS"] = False
+        self._cmake.definitions["CASS_DEBUG_CUSTOM_ALLOC"] = False
+        self._cmake.definitions["CASS_INSTALL_HEADER_IN_SUBDIR"] = self.options.install_header_in_subdir
+        self._cmake.definitions["CASS_INSTALL_PKG_CONFIG"] = False
+
+        if self.options.use_atomic == "boost":
+            self._cmake.definitions["CASS_USE_BOOST_ATOMIC"] = True
+            self._cmake.definitions["CASS_USE_STD_ATOMIC"] = False
+
+        elif self.options.use_atomic == "std":
+            self._cmake.definitions["CASS_USE_BOOST_ATOMIC"] = False
+            self._cmake.definitions["CASS_USE_STD_ATOMIC"] = True
+        else:
+            self._cmake.definitions["CASS_USE_BOOST_ATOMIC"] = False
+            self._cmake.definitions["CASS_USE_STD_ATOMIC"] = False
+
+        self._cmake.definitions["CASS_USE_OPENSSL"] = self.options.with_openssl
+        self._cmake.definitions["CASS_USE_STATIC_LIBS"] = False
+        self._cmake.definitions["CASS_USE_ZLIB"] = self.options.with_zlib
+        self._cmake.definitions["CASS_USE_LIBSSH2"] = False
+
+        # FIXME
```

Comment:
```suggestion
        # FIXME
```

FIXME what? Please, add more detail on that tag.

Reply:
[Fixed](https://github.com/conan-io/conan-center-index/pull/3110/commits/79bf53e48310fa3a088ca65ab170ef510bf0caa8)